### PR TITLE
PBFT - late joining node 1/????

### PR DESCRIPTION
**Pull request by ashamis**
in the ePBFT configuration when a node join late we need to pass a certificate and also initialize ePBFT on the late joining node.

### CODE REVIEW

File: src/consensus/pbft/pbftconfig.h [hunk 1]
```diff
@@ -102,9 +119,16 @@ namespace pbft
 
       auto frontend = handler.value();
 
+      std::string cert((char*)request->cert(), request->cert_size);
```
Comment (achamayou) [inline src/consensus/pbft/pbftconfig.h]: Remove?

**Comment by cimetrics** [comment]
images

File: src/enclave/enclavetypes.h [hunk 2]
```diff
@@ -71,6 +71,19 @@ namespace enclave
       actor(actor_)
     {}
 
+    // Constructor used for forwarded RPC
```
Comment (jumaffre) [inline src/enclave/enclavetypes.h]: Update comment here

File: src/node/nodestate.h [hunk 3]
```diff
@@ -362,8 +362,10 @@ namespace ccf
           self = res.id;
 #ifndef PBFT
```
Comment (jumaffre) [inline src/node/nodestate.h]: Minor: Let's keep it consistent with the code in `create()`:
```
#ifdef PBFT
          setup_pbft();
#else
          setup_raft();
#endif
```

**Comment by cimetrics** [comment]
images

**Comment by cimetrics** [comment]
images

**Pull request merged**